page-templates/flexible.php

```diff
@@ -1,11 +1,17 @@
 <?php /* Template Name: Flexible */
 
-use \Timber\Timber;
-
-get_header();
+use Timber\Timber;
+use LeanNs\Assets;
 
 $data = Timber::get_context();
 $data['post'] = \LeanNs\Patterns::get_post();
+
+if ( 'images' === $data['post']->acf['hero']['hero_background']['type'] ) {
```

Comment:
It should always be there as it relates to the ACF fields defined for this template. If someone updates the ACF (e.g. removes the hero from this page) then it's probably best to throw an exception so they remember to remove this code too. What do you think?